en/02_Developer_Guides/00_Model/11_Scaffolding.md

```diff
@@ -466,6 +479,62 @@ class MyDataObject extends DataObject
 }
 ```
 
+#### Using `WithinRangeFilter` {#searchable-fields-withinrangefilter}
+
+If you want users to be able to filter by a field using a range, specify the [`WithinRangeFilter`](api:SilverStripe\ORM\Filters\WithinRangeFilter). This works out of the box with the numeric, date, datetime, and time fields that come in Silverstripe framework.
+
+```php
+namespace App\Model;
+
+use SilverStripe\ORM\DataObject;
+SilverStripe\ORM\Filters\WithinRangeFilter
+
+class MyDataObject extends DataObject
+{
+    // ...
+    private static array $db = [
+        'Price' => 'Currency',
```

Comment:
The currency filter currently seems very busted, even with the framework. Change this example use an Int instead so when people copy it they don't think that they've done something wrong

Reply:
I'd rather just fix the field. If it's just the validation thing, I'll pull that card into in progress and fix it. If there are more problems, please open cards for them.

Reply:
Noted on https://github.com/silverstripe/silverstripe-framework/pull/11602#issuecomment-2661684309 it's related to the framework PR, so should be fixed there